lib/site_prism/page.rb

```diff
@@ -64,12 +64,8 @@ def load(expansion_or_html = {}, &block)
     end
 
     def displayed?(*args)
-      expected_mappings = args.last.is_a?(::Hash) ? args.pop : {}
-      seconds = !args.empty? ? args.first : Capybara.default_max_wait_time
-      raise SitePrism::NoUrlMatcherForPageError unless url_matcher
-
       begin
-        Waiter.wait_until_true(seconds) { url_matches?(expected_mappings) }
+        Waiter.wait_until_displayed(*args, url_matcher) { url_matches?(*args) } 
```

Comment:
passing the extra var through is un-needed, everything is auto-loaded from the raw DSL.

If you find this causes issues, require the requisite file in the Waiter class

Reply:
@luke-hill `url_matches?` is `private` within the page class so it causes an issue in that respect

Reply:
But it will always be calling it from within another method, so it shouldn't cause any issues. If loading is a problem, require the file.